.github/workflows/build-timestamped-master.yml

```diff
@@ -40,7 +40,7 @@ jobs:
       - name: Generate CodeCov Report
         uses: codecov/codecov-action@v2
       - name: Upload Artifact
-        uses: actions/upload-artifact@v2
+        uses: actions/upload-artifact@v4
```

Comment:
Shall we update other actions in the same workflow as well? and check the other workflows too?
This could be a potential future task.

Reply:
Yeah let's do that.

Reply:
Updated in 6f7b17f3c51db5d067caf48b51c77c6f710204fb